### PR TITLE
spacing of ref-qualifiers

### LINKED ISSUE #5308: How to space ref qualifiers
Should we have a convention for how to use whitespace in ref qualifiers in the library?

```
void f() const &;
void f() const&;
void f() const &&;
void f() const&&;
```

These qualifiers are still relatively rare, so we don't really have a weight of precedence here. I'm inclined to have no space between the cv and the ref qualifier (`const&`). If we want to fix a style, we should record it in the wiki.

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #5308

### CODE REVIEW

**Comment by jensmaurer** [comment]
@tkoeppe, here you are

**Comment by tkoeppe** [comment]
Beautiful, thank you!

File: source/basic.tex [hunk 1]
```diff
@@ -965,7 +965,7 @@
   void h(this X&, int);
   void h(int) &&;               // OK, another overload
   void j(this const X&);
-  void j() const&;              // error: redeclaration
+  void j() const &;             // error: redeclaration
```
Comment (JohelEGP) [inline source/basic.tex:968]: This is the only occurrence in core wording (in an example). Why not check only the library sources?

Comment (jensmaurer) [reply source/basic.tex:968]: Sure. Done.

**Comment by jensmaurer** [comment]
@tkoeppe, want to merge this?

**Pull request merged**